src/matching/Matching_generic.mli

```diff
@@ -9,7 +9,10 @@ type tin = {
   config : Rule_options.t;
   deref_sym_vals : int;
       (** Counts the number of times that we "follow" symbollically propagated
-    * values. This is bound to prevent potential infinite loops. *)
+       * values. This is bound to prevent potential infinite loops. *)
+  inside_function : bool;
```

Comment:
Shouldn't the docstrings be in the `.mli` ?

Reply:
You're right! Thanks for pointing this out. From @brandonspark and @aryx 's input, I think I'm actually moving away from putting these fields in `tin`, but your feedback still applies anyway, so I'll keep that in mind.